__tests__/app.test.js

```diff
@@ -71,7 +71,8 @@ describe('articles', () =>
                     author: "butter_bridge",
                     body: "I find this existence challenging",
                     created_at: "2020-07-09T20:11:00.000Z",
-                    votes: 100
+                    votes: 100,
+                    comment_count: "11"
```

Comment:
It's not best practise to change previous tests (which you've had to do in this case as you've used toEqual rather than objectContaining). It would be have been best to use objectContaining originally, then you could add a separate test to check if the returned article also contains comment_count.